src/web/landing/src/Flow/Website/Controller/DefaultController.php

```diff
@@ -2,120 +2,62 @@
-            ],
-            [
-                'url' => 'https://github.com/peter279k',
-                'avatar' => 'https://avatars.githubusercontent.com/u/9021747?s=60&amp;v=4',
-                'name' => 'peter279k',
-            ],
-            [
-                'url' => 'https://github.com/saulblake',
-                'avatar' => 'https://avatars.githubusercontent.com/u/582274?s=60&amp;v=4',
-                'name' => 'saulblake',
-            ],
-            [
-                'url' => 'https://github.com/jpiatko',
-                'avatar' => 'https://avatars.githubusercontent.com/u/80686947?s=60&amp;v=4',
-                'name' => 'jpiatko',
-            ],
-            [
-                'url' => 'https://github.com/drupol',
-                'avatar' => 'https://avatars.githubusercontent.com/u/252042?s=60&amp;v=4',
-                'name' => 'drupol',
-            ],
-            [
-                'url' => 'https://github.com/Wirone',
-                'avatar' => 'https://avatars.githubusercontent.com/u/600668?s=60&amp;v=4',
-                'name' => 'Wirone',
-            ],
-            [
-                'url' => 'https://github.com/mleczakm',
-                'avatar' => 'https://avatars.githubusercontent.com/u/3474636?s=60&amp;v=4',
-                'name' => 'mleczakm',
-            ],
-            [
-                'url' => 'https://github.com/flavioheleno',
-                'avatar' => 'https://avatars.githubusercontent.com/u/471860?s=60&amp;v=4',
-                'name' => 'flavioheleno',
-            ],
-            [
-                'url' => 'https://github.com/voku',
-                'avatar' => 'https://avatars.githubusercontent.com/u/264695?s=60&amp;v=4',
-                'name' => 'voku',
-            ],
-            [
-                'url' => 'https://github.com/FunkyOz',
-                'avatar' => 'https://avatars.githubusercontent.com/u/26649880?s=60&amp;v=4',
-                'name' => 'FunkyOz',
-            ],
-        ];
+        $factory = new Psr17Factory();
+        $client = new Client($factory, $factory);
+
+        df()
+            ->read(
+                new PsrHttpClientDynamicExtractor($client, $this->contributorsUrlFactory)
+            )
+            // Extract response
+            ->withEntry('unpacked', ref('response_body')->jsonDecode())
+            ->select('unpacked')
+            // Extract data as rows & columns
+            ->withEntry('data', ref('unpacked')->expand())
+            ->withEntry('data', ref('data')->unpack())
+            ->renameAll('data.', '')
+            ->drop('unpacked', 'data')
+            // Filter out bots & limit the end list
+            ->filter(not(ref('login')->endsWith(lit('[bot]'))))
+            ->filter(not(ref('login')->equals(lit('aeon-automation'))))
+            ->limit(24)
+            // Save with overwrite
+            ->mode(SaveMode::Overwrite)
+            ->write(to_json($this->path))
+            // Execute
+            ->run();
```

Comment:
I like how flow is used to generate flow landing page contributors ❤️

Reply:
but technically speaking you dont even need to write it anywhere, you can literally load it to memory and dump that memory to twig directly